src/config.ts

```diff
@@ -18,6 +18,7 @@ const SCRIPT_TYPES_WEIGHTED = {
   bundle: 100,
 } as {[type in ScriptType]: number};
 
+// FUTURE: Remove 'proxy' on next major release
 type ScriptType = 'proxy' | 'mount' | 'run' | 'build' | 'bundle';
```

Comment:
"proxy" is safe to remove now, since we're splicing any "proxy" scripts out of the `config.scripts` array. This should also help you find the parts of `dev` referencing this type of script that can also be cleaned up now.